esp/services/ws_store/ws_storeService.cpp

```diff
@@ -120,7 +120,7 @@ IEspStore* CwsstoreEx::loadStoreProvider(const char* instanceName, const char* l
 bool CwsstoreEx::onCreateStore(IEspContext &context, IEspCreateStoreRequest &req, IEspCreateStoreResponse &resp)
 {
     const char *user = context.queryUserId();
-    m_storeProvider->createStore(req.getType(), req.getName(), req.getDescription(), new CSecureUser(user, nullptr));
+    resp.setSuccess(m_storeProvider->createStore(req.getType(), req.getName(), req.getDescription(), new CSecureUser(user, nullptr)));
```

Comment:
@rpastrana you may need to check the client version before setSuccess().

Reply:
I though about that, but what's the drawback in setting the field always?